client/src/components/viewers/types.ts

```diff
@@ -0,0 +1,13 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+import { Translatable } from 'megashark-lib';
+
+export interface FileControlsDropdownItemContent {
+  label: Translatable;
+  callback?: (arg?: any) => Promise<any>;
```

Comment:
Why would you need arguments? You're calling the callback yourself and you don't pass any.